src/infrastructure/figma/figma-service.ts

```diff
@@ -54,66 +48,51 @@ export class FigmaService {
 	 *
 	 * @throws {UnauthorizedFigmaServiceError} Not authorized to access Figma.
 	 */
-	fetchDesignById = async (
+	getDesign = async (
 		designId: FigmaDesignIdentifier,
 		user: ConnectUserInfo,
 	): Promise<AtlassianDesign> =>
 		this.withErrorTranslation(async () => {
-			const { accessToken } = await figmaAuthService.getCredentials(user);
+			const credentials = await figmaAuthService.getCredentials(user);
 
-			if (designId.nodeId) {
```

Comment:
**note:** Encapsulated the code into separate methods: `getDesignForFile` and `getDesignForNode`.